ui/src/authors/containers/DetailPageContainer/DetailPageContainer.tsx

```diff
@@ -283,6 +289,7 @@ const mapStateToProps = (state: RootStateOrAny) => ({
     'initialTotal',
   ]),
   isCatalogerLoggedIn: isCataloger(state.user.getIn(['data', 'roles'])),
+  superuser: isSuperUser(state.user.getIn(['data', 'roles'])),
```

Comment:
to be more consistent:
```suggestion
  isSuperuserLoggedIn: isSuperUser(state.user.getIn(['data', 'roles'])),
```

Reply:
@karolina-siemieniuk-morawska please not the camel case :)